### ISSUE TITLE
Click event for clear action should not bubble up

### ISSUE THREAD

**Issue by JZubero** [vueform/multiselect#285]
When clicking the clear icon, the dropdown shows shortly before the selection is cleared and dropdown closes again.

This could be avoided with the `.stop` modifier:

```
    <!-- Clear -->
    <slot v-if="hasSelected && !disabled && canClear && !busy" name="clear" :clear="clear">
      <span
        tabindex="0"
        role="button"
        aria-label="❎"
        :class="classList.clear"
        @click.stop="clear"
        @keyup.enter="clear"
      ><span :class="classList.clearIcon"></span></span>
    </slot>
```

**Comment by adamberecz** [comment]
Thanks, fixed in `2.5.7`.

**Comment by JZubero** [comment]
@adamberecz Confirmed :heavy_check_mark: Thanks for this.
